server/utils/dateUtils.test.ts

```diff
@@ -375,4 +376,39 @@ describe('DateFormats', () => {
       expect(result.formattedDate).toBeTruthy()
     })
   })
+
+  describe('dateIsBlank', () => {
+    it('returns false if the date is not blank', () => {
+      const date: ObjectWithDateParts<'field'> = {
+        'field-day': '12',
+        'field-month': '1',
+        'field-year': '2022',
+      }
+
+      expect(dateIsBlank(date, 'field')).toEqual(false)
+    })
+
+    it('returns true if the date is blank', () => {
```

Comment:
👋 Would it make sense to also cover the case where only some inputs were provided?

Reply:
👍